spec/models/education_spec.rb

```diff
@@ -1,5 +1,56 @@
 require 'rails_helper'
 
 RSpec.describe Education, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'has a valid factory' do
+    expect(FactoryGirl.build(:education)).to be_valid
+  end
+
+  describe 'converts date to strftime' do
+    education = FactoryGirl.build(:education)
+    it 'converts education start date to strftime' do
+      expect(education.pretty_date(education.start_date)).to eq '01/01/2001'
```

Comment:
If `pretty_date` is a method on the `Education` model, you don't need to pass in the `education.start_date`. `start_date` is also an instance method on the `Education` model so it's available to you there. You should be able to just do `education.pretty_date` and it return the formatted date. Take a look at the implementation in the `Education` model and make sure it was implemented correctly. It looks like it wasn't.